src/core_ocean/mode_init/Registry.xml

```diff
@@ -22,6 +22,20 @@
 	<!--**********************************-->
 	<!-- Namelists for init run mode only -->
 	<!--**********************************-->
+	<nml_record name="partial_bottom_cells" mode="init">
+		<nml_option name="config_alter_ICs_for_pbcs" type="logical" default_value=".false." units="unitless"
+					description="If true, initial conditions are altered according to the config_pbc_alteration_type flag."
+					possible_values=".true. or .false."
+		/>
+		<nml_option name="config_pbc_alteration_type" type="character" default_value="full_cell" units="unitless"
```

Comment:
I'm confused as to why these additions for removal of code... can you please clarify?

Reply:
I see where they were removed above-- thanks.